distribution/docs/src/main/jdocs/content/_plugins/post-ingest-plugin-intro-contents.adoc

```diff
@@ -0,0 +1,36 @@
+:type: pluginIntro
+:status: published
+:title: Post-Ingest Plugins
+:link: _post_ingest_plugins
+:summary: Perform actions after ingest is completed.
+:plugintypes: postingest
+:order: 05
+
+.Query Plugin Flow
```

Comment:
❓ The `title`, `link` and last paragraph refer to post-ingest plugins, but the rest of this section is about query plugins.